.github/workflows/deploy.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies
-        run: yarn install --frozen-lockfile --ignore-engines --prefer-offline
+        run: yarn install --immutable
```

Comment:
This is what Studio uses. 

immutable replaces frozen-lockfile;  --ignore-engines and --prefer-offline are no longer necessary.